### ISSUE TITLE
Where can one find the documentation about how to perform a release

### ISSUE THREAD

**Issue by hiddenalpha** [swisspost/vertx-redisques#176]
I searched through the README but couldn't find anything useful. Neither an explanation, nor any link to the corresponding doc.

**Comment by mcweba** [comment]
You are right, currently there is no documentation about this. We should at it to the README. Meanwhile, to create a new release, make a PullRequest to the `master` branch. This will trigger the release. After that, you can select the new created tag in the release section of github.

**Comment by hiddenalpha** [comment]
Do I have to merge the PR in the end?

**Comment by mcweba** [comment]
> Does one have to merge the PR in the end, after the builds is ok?

Yes I forgot to mention this. You create the PR to the `master` branch. After the build went sucessfully through, you merge the PR and this will then trigger the release build.
